### PR TITLE
53 autonomous code

### LINKED ISSUE #53: Start On Autonomous
Need to review strategies and pick a few so we can code the robot accordingly. Use color team switches. Done when we have good Autonomous codes.

### PR DESCRIPTION
**Pull request by Glorfindel29**
Rady for regionals.

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Autonomous.java [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+        // breaks the motor
+        screw.setZeroPowerBehavior(DcMotor.ZeroPowerBehavior.BRAKE);
+        screw.setPower(0);
+        screw.setMode(DcMotor.RunMode.STOP_AND_RESET_ENCODER);
+        screwLevel = 0;
+    }
+
+    private void driveScrewDown(double screwTarget, double speed)
+    {
```
Comment (BrendanFish) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Autonomous.java:258]: THE CURLY BRACES

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java [hunk 2]
```diff
@@ -1,130 +1,342 @@
...
+        driveScrewDown(10000, 1, telemetry);
+
+        // Make sure Screw will go the right way
+        screw.setDirection(DcMotor.Direction.REVERSE);
     }
 
-    public void handleScrew() {
-        if(raiseTower){
-//            if(highSensor.getState()){
-//                screw.setPower(0);
```
Comment (BrendanFish) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java]: lots of stuff commented out. If we don't want it, I would delete it.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java [hunk 3]
```diff
@@ -1,130 +1,342 @@
...
+    {
+        double uBarPower;
+        double driveUBar = gamepad.right_stick_y;
+        uBarPower = Range.clip(driveUBar, -1, 1);
+//        if (!(uBarPower == 0))
+//        {
+//            uBar.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
+//            uBar.setPower(uBarPower);
+//        }
+//        else
```
Comment (BrendanFish) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java:288]: Same here, big amounts of code, if we're not using it delete it.

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/localizers/CombinedLocalizer.java [hunk 4]
```diff
@@ -162,6 +162,7 @@ public void displayTelemetry(Telemetry telemetry) {
         telemetry.addData("heading","%.1f",heading);
         telemetry.addData("Brendan's heading","%.1f",smartAngleError(heading, 0));
```
Comment (BrendanFish) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/localizers/CombinedLocalizer.java:163]: what is "Brendan's Heading"? I don't know the robot's heading. Better label

**Review by BrendanFish — CHANGES_REQUESTED** [review]
small things to fix

**Review by Glorfindel29 — COMMENTED** [review]
Aprove it

**Pull request merged**